src/components/Navigation/Navigation.jsx

```diff
@@ -37,32 +32,26 @@ export default function Navigation(){
 		}
 	];
 
-
-	const links       = linkData.map(RENDER.item);
-	const ariaOpen    = open.toString();
+	const links      	  = linkData.map(RENDER.item);
+	const ariaExpanded    = expanded.toString();
```

Comment:
```suggestion
	const expanded    = expanded.toString();
```
once we've removed the state we can put this back to its non-namespaced variant